content/hardware/build_instructions.md

```diff
@@ -733,27 +732,23 @@ In this section, we will assemble the back cover, mount the Jetson Nano, and att
 
     {{< figure src="/hardware/build_instructions/12_02_insert_jetson_mount_hex_nuts.png" caption="Fig. 12.2" width="800">}}
 
-7. Tilt the back cover so that the hex nuts from the previous step stay in place. Simultaneously align the Jetson Nano with these hex nuts, and secure its mounting screws to the BCT. (Fig. 12.3)
+7. Tilt the back cover so that the hex nuts from the previous step stay in place. Simultaneously align the Jetson Nano/Xavier NX with these hex nuts, and secure its mounting screws to the BCT. (Fig. 12.3)
 
     {{< figure src="/hardware/build_instructions/12_03_mount_jetson_nano.png" caption="Fig. 12.3" width="800">}}
 
-8. Now that the Jetson Nano has been mounted, put a pin jumper on the J48 header, as shown in Fig 12.4, so that the Jetson Nano draws power from the DC barrel jack.
-
-    {{< figure src="/hardware/build_instructions/12_04_place_nano_jumper.png" caption="Fig. 12.4" width="800">}}
-
-9. Next, we will connect the push button to the Jetson Nano's breakout pins. Note that in the following description, all pins refer to pins found on the Jetson Nano's J41 header, and that we will refer to specific jumper cables as defined in the push button installation section. Connect jumper cable B to Pin 31, which serves as an input GPIO pin. Connect jumper cable C to Pin 1, which connects the pull-up resistor to 3.3V. Finally, connect jumper cable D to Pin 39, which serves as a ground connection when the button is pressed.
+8. Next, we will connect the push button to the Jetson Nano's breakout pins. Note that in the following description, all pins refer to pins found on the Jetson Nano's J41 header, and that we will refer to specific jumper cables as defined in the push button installation section. Connect jumper cable B to Pin 31, which serves as an input GPIO pin. Connect jumper cable C to Pin 1, which connects the pull-up resistor to 3.3V. Finally, connect jumper cable D to Pin 39, which serves as a ground connection when the button is pressed. Use the same pins for the Xavier NX.
```

Comment:
Was the step about placing a pin jumper on J48 accidentally removed?

Reply:
No, I removed that on purpose because the Jetson Nano now comes with the pin jumper already in place, there's no need to manually place it.